.gitignore

```diff
@@ -1,2 +1 @@
 /bazel-*
-.bazelrc.local
```

Comment:
Why? We do need this line here. A `.bazelrc.local` should not be checked in.

Reply:
Good catch. Was a WIP thing that slipped through (because I added the symlinks to `tutorial` and `examples`).